pkgs/by-name/xp/xppen/package.nix

```
@@ -0,0 +1,53 @@
+{ stdenv
+, lib
+, fetchzip
+, libusb1
+, autoPatchelfHook
+, qt5
+, libGL
+}:
+stdenv.mkDerivation rec {
+  pname = "xppen";
+  version = "3.4.9";
+
+  src = fetchzip {
+    extension = "tar.gz";
+    url = "https://www.xp-pen.com/download/file.html?id=1936&pid=990&ext=gz";
+    hash = "sha256-A/dv6DpelH0NHjlGj32tKv37S+9q3F8cYByiYlMuqLg=";
+  };
+
+  nativeBuildInputs = [ autoPatchelfHook ];
+
+  buildInputs = [ qt5.full libusb libGL ];
+
+  dontConfigure = true;
+  dontBuild = true;
+  dontCheck = true;
+
+  installPhase = ''
+        runHook preInstall
+        mkdir -p $out/bin $out/share
+        cp -r App/usr/lib/pentablet/. $out/bin
+        rm $out/bin/PenTablet.sh
+        cp -r App/usr/share/. $out/share
+        rm -r $out/bin/lib
+        cp -r App/lib $out/lib
+        runHook postInstall
+  '';
+
+  postInstall = ''
+        substituteInPlace $out/share/applications/xppentablet.desktop \
+          --replace "/usr/lib/pentablet/PenTablet.sh" "PenTablet" \
+          --replace "/usr/share/icons/hicolor/256x256/apps/xppentablet.png" "xppentablet"
+  '';
```

We don't really have that many style guidelines yet (waiting for [RFC 166](https://github.com/NixOS/rfcs/pull/166)), but 2 space indents is part of the ones we have: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#syntax

```suggestion
  installPhase = ''
    runHook preInstall
    mkdir -p $out/bin $out/share
    cp -r App/usr/lib/pentablet/. $out/bin
    rm $out/bin/PenTablet.sh
    cp -r App/usr/share/. $out/share
    rm -r $out/bin/lib
    cp -r App/lib $out/lib
    runHook postInstall
  '';

  postInstall = ''
    substituteInPlace $out/share/applications/xppentablet.desktop \
      --replace "/usr/lib/pentablet/PenTablet.sh" "PenTablet" \
      --replace "/usr/share/icons/hicolor/256x256/apps/xppentablet.png" "xppentablet"
  '';
```